boards/arm/mimxrt1050_evk/mimxrt1050_evk.yaml

```diff
@@ -26,3 +26,4 @@ supported:
   - usb_device
   - kscan:touch
   - watchdog
+  - adc
```

Comment:
Please update the documentation for these boards as well including ADC in supported feature list and the pins used.

Reply:
add adc description to board doc